src/api/forms/service.js

```diff
@@ -89,9 +96,13 @@ export async function listForms() {
 export async function getForm(formId) {
   const document = await formMetadata.get(formId)
 
-  if (document) {
-    return mapForm(document)
+  if (!document) {
+    throw Boom.notFound(
+      `Form ${formId} does not exist, so the definition cannot be updated.`
+    )
```

Comment:
Nice one, shall we do the same for `getFormBySlug`?

Worth mentioning too for consistency:

* **draft-form-definition-repository.js** – Throws `Boom.notFound()` itself
* **form-metadata-repository.js** – Only the calling code throws `Boom.notFound()`

Where would you prefer the Boom errors be thrown?

Reply:
Oh! Did we copy the wrong error text for this one?

Think it should still be the old message, perhaps with ~`id`~ `ID` now

```suggestion
  if (!document) {
    throw Boom.notFound(`Form with ID '${formId}' not found`)
```